app/scripts/background.js

```diff
@@ -454,74 +469,69 @@ function saveTimestamp() {
  * @returns {Promise} Setup complete.
  */
 async function initialize() {
-  try {
```

Comment:
the error now bubbles up to the top `init` function, where it gets handle there.